src/main/java/calculator/model/CalcNumber.java

```diff
@@ -0,0 +1,33 @@
+package calculator.model;
+
+public class CalcNumber {
+    public int value;
+    private static final int DEFAULT_CALC_VALUE = 0;
```

Comment:
아래의 코드로 개선해 보면 어떨까요? 불필요한 객체 생성을 피할 수 있어요.

```suggestion
    public static final CalcNumber DEFAULT_CALC_VALUE = new CalcNumber(0);
```